src/main/java/com/minecolonies/network/messages/TownHallRenameMessage.java

```diff
@@ -5,19 +5,17 @@
 import com.minecolonies.colony.ColonyManager;
 import com.minecolonies.colony.ColonyView;
 import io.netty.buffer.ByteBuf;
+import net.minecraft.entity.player.EntityPlayerMP;
 import net.minecraftforge.fml.common.network.ByteBufUtils;
 import net.minecraftforge.fml.common.network.simpleimpl.IMessage;
-import net.minecraftforge.fml.common.network.simpleimpl.IMessageHandler;
-import net.minecraftforge.fml.common.network.simpleimpl.MessageContext;
 import org.jetbrains.annotations.NotNull;
-import org.jetbrains.annotations.Nullable;
 
-public class TownHallRenameMessage implements IMessage, IMessageHandler<TownHallRenameMessage, IMessage>
+public class TownHallRenameMessage extends AbstractMessage<TownHallRenameMessage, IMessage>
 {
+    private static final int MAX_NAME_LENGTH  = 25;
+    private static final int SUBSTRING_LENGTH = MAX_NAME_LENGTH - 1;
     private int    colonyId;
     private String name;
-    private static final int MAX_NAME_LENGTH = 25;
-    private static final int SUBSTRING_LENGTH = MAX_NAME_LENGTH - 1;
 
     public TownHallRenameMessage() {}
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) It is a good practice to call super() in a constructor [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ACallSuperInConstructor)